ui/src/dataExplorer/components/SaveAsButton.tsx

```diff
@@ -76,7 +76,9 @@ class SaveAsButton extends PureComponent<Props, State> {
               {saveAsOption === SaveAsOption.Dashboard && (
```

Comment:
We could use a getter here since we never need these both at the same time and there will be more options soon.